.claude/commands/gap.md

```
@@ -0,0 +1,45 @@
+---
+description: Analyze conversation context for unaddressed items and gaps
+argument-hint: [optional additional info]
+---
+
+## General Guidelines
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)